cmd/mt-index-migrate/main.go

```diff
@@ -22,6 +22,7 @@ var (
 	dstCassAddr     = flag.String("dst-cass-addr", "localhost", "Address of cassandra host to migrate to.")
 	srcKeyspace     = flag.String("src-keyspace", "raintank", "Cassandra keyspace in use on source.")
 	dstKeyspace     = flag.String("dst-keyspace", "raintank", "Cassandra keyspace in use on destination.")
+	idxTable        = flag.String("idx-table", "metric_idx", "index table name in Cassandra")
```

Comment:
wouldn't we want a src-table-name and dst-table-name flag here?

Reply:
Absolutely! My bad. Fixed in new commit.